File: catorce ejercicios/PHP2/Php/ejemplo.1.php
<!-- Diseñar un algoritmo que dada la edad de una persona indique si es MAYOR o MENOR de edad. -->


<form method="POST" action="">
        <label for="edad">Ingrese su edad:</label>
        <input type="number" id="edad" name="edad" required>
        <input type="submit" value="Enviar">
</form>

<?php
if ($_SERVER["REQUEST_METHOD"] === "POST") {
    $edad = $_POST['edad'];

    if (is_numeric($edad) && $edad > 0 && $edad <= 100) {
        if ($edad >= 18) {
            echo "Usted tiene " . $edad . " años, eres mayor de edad.";
        } else {
            echo "Usted tiene " . $edad . " años, eres menor de edad.";
        }
    } else {
        echo "ERROR: La edad debe ser un número mayor a 0 y menor o igual a 100.";
    }
}
?>

File: catorce ejercicios/PYTHON2/ejemplo.1.py
#Diseñar un algoritmo que dada la edad de una persona indique si es MAYOR o MENOR de edad.


print("Ejemplo 1")

edad = int (input ("Ingrese su edad, por favor: "))

if edad == 0 or edad > 100:
    print ("Edad incorrecta")
        
elif edad >= 18:
    print ("Usted es mayor de edad ",edad)

elif edad <= 17:
    print ("Usted es menor de edad ",edad)